tempodb/backend/block_meta.go

```diff
@@ -22,6 +22,8 @@ type BlockMeta struct {
 	StartTime       time.Time `json:"startTime"`
 	EndTime         time.Time `json:"endTime"`
 	TotalObjects    int       `json:"totalObjects"`
+	TotalSize       uint64    `json:"totalSize"`
+	FinalSize       uint64    `json:"finalSize"`
```

Comment:
Why do we need two sizes in the block meta? Let's just use the compressed size?

Reply:
Sounds like the consensus is that storing the uncompressed data size isn't valuable.  Removed.